win,tty: Exposing uv_set_vterm_state (#2129)
libuv/libuv: docs/src/tty.rst

          UV_TTY_MODE_IO
      } uv_tty_mode_t;

.. c:type:: uv_tty_vtermstate_t

    Console virtual terminal mode type:

    ::

      typedef enum {
          /* unchecked */
          UV_TTY_AUTODETECT,
          /* Legacy conhost */
          UV_TTY_LEGACY,
          /* Modern conhost */
          UV_TTY_VTP,
          /* ConEmu, mintty, etc. */

>           uv_TTY_ANSI

Reviewer: ```suggestion
          UV_TTY_ANSI
```

    Gets the current Window size. On success it returns 0.

.. c:function:: int uv_tty_set_vterm_state(uv_tty_t* handle, uv_vtermstate_t state)

>     Set the Console using the specified virtual terminal mode.

Reviewer 2: ```suggestion
    Set the Console to use the specified virtual terminal mode.
```

    Gets the current Window size. On success it returns 0.

.. c:function:: int uv_tty_set_vterm_state(uv_tty_t* handle, uv_vtermstate_t state)

    Set the Console to use the specified virtual terminal mode.

    This function is only implemented on Windows.

>     .. versionadded:: 1.25.0

Reviewer 2: ```suggestion

    .. versionadded:: 1.25.0
```

I don't know if the extra space is required for the markdown format, but the other places in the file seem to have it.

    Gets the current Window size. On success it returns 0.

> .. c:function:: int uv_tty_set_vterm_state(uv_tty_t* handle, uv_vtermstate_t state)

Reviewer: ```suggestion
.. c:function:: int uv_tty_set_vterm_state(uv_tty_t* handle, uv_tty_vtermstate_t state)
```

>     Set the Console to use the specified virtual terminal mode.

Reviewer: ```suggestion
    Set the console to use the specified virtual terminal mode.
```

    This function is only implemented on Windows.

>     .. versionadded:: 1.25.0

Reviewer: This would be 1.31.0 now.

Reviewer: Friendly reminder that this still needs to be updated.

>     .. versionadded:: 1.31.0

Author: Need to fix before merging.
